.github/ISSUE_TEMPLATE/feature-request.yml

```diff
@@ -0,0 +1,52 @@
+name: Feature request
+description: Request a new feature or improvement
+labels: ["feature"]
+
+body:
+  - type: textarea
+    id: summary
+    attributes:
+      label: Summary
+      description: A description of what you want implemented.
+      placeholder: "Example: The ability to install projects in editable mode."
+    validations:
+      required: true
+
+  - type: textarea
+    id: problem
+    attributes:
+      label: Why?
+      description: The reason you want this implemented.
+      placeholder: "Example: It will make iterating during development faster."
+    validations:
+      required: true
+
+  - type: textarea
+    id: ideal-implementation
+    attributes:
+      label: Ideal implementation
+      description: How you think this should be implemented, if you have an idea.
+      placeholder: "Example: A `-e` command line flag that enables editable mode."
+
+  - type: textarea
+    id: additional-info
+    attributes:
+      label: Additional information
+      description: If there's anything else you feel is relevant, include it here.
+
+  - type: checkboxes
+    id: contribution
+    attributes:
+      label: Contribution
+      options:
+        - label: I wish to contribute this feature.
```

Comment:
I know this is from analytix, but I have personally kinda gonna back on this recently. It would probably be easier to assign issues rather than have this checkbox.